### DISCUSSION TITLE
YouTubeRebornPlus Discussion

### DISCUSSION THREAD

**Discussion by arichornlover** [arichornlover/YouTubeRebornPlus#1] [General]
## Read Here ⚠️
As you may have known, this repository will get Archived when I feel like it needs to.
The original Repository known as https://github.com/qnblackcat/CercubePlus got taken down. And that’s honestly the reason why I’m archiving this to prevent having the same fate as qnblackcat. Archiving it is best for downloading the latest release and no consequences in the future but read below to see what else it can do. I will do requests if you read below
## Archive Changes:
**No Commits to this Repository**

**No New Releases Unless if there’s issues.**

**Repository labeled as “Archived”**

don’t be sad yet y’all. I will still do releases of CercubePlus if it’s really important or requested by y’all!
## Requests you can do ✅
**Request New Release of CercubePlus (only do this if YouTube Version stops working, Cercube Breaking, Crashing or really want New Features added from uYouPlus or any other source)

## Pull Requests (Unfortunately when it’s Archived it may not be possible so put the code in Issues so I can change it in the future.)

**Comment by arichornlover** [comment]
Also y’all I’m sorry about the branches! The branches are being modified by me and since I’m doing some tests. I’m doing the right thing by gladly moving stuff in a branch that may not work with CercubePlus Yet.

**Reply by nexus8x** [reply]
![C8F6A327-8EE4-47E0-BA76-91B99B824F6A](https://user-images.githubusercontent.com/102811744/210455460-4a21c813-8a1d-418c-b888-e978acc863b2.png)
![DC4EDB30-8A0E-4234-BDBD-1F1E599ED2BD](https://user-images.githubusercontent.com/102811744/210455465-26634f0f-1cef-4345-9935-b65fcd8ad74e.png)
Hi Bro...You can add hidden options like iSponsor,Cercube, uYou....like iApp's YTplus?  It is also built from uYou+..

**Reply by arichornlover** [reply]
I don’t know what you said but I’ve been adding a ton of new options to CercubePlus for the past 2 weeks, this release won’t have some of the options shown from the image you sent. But maybe I can hide the SubBar if I got enough time.

**Reply by nexus8x** [reply]
I mean.. can you add options like Remove iSponsorbkock, Cercube button from Navigation bar?  or Hide uYou tab button from tab bar...?..It will make the screen clean and like a real youtube premium version..

**Reply by arichornlover** [reply]
> Hi Bro...You can add hidden options like iSponsor,Cercube, uYou....like iApp's YTplus? It is also built from uYou+..

wait. looking at your screenshot, can you take me to the source code of this tweak? because I may not be able to actually pull it off and actually see if it worked

also I added an option to hide the Sub Bar thanks to uYou+'s recent changes today

**Reply by nexus8x** [reply]
https://twitter.com/dayanch96?s=21&t=76bsW58BSoEy8Lwdirc4Qw
This is his twitter..he is a YT+ developer..also uYou+based.. I guess you know him too.

**Reply by arichornlover** [reply]
Although I don’t know them but I will ask Dayanch about the options “Hide iSponsorBlock Button” & “uYou Tab” Buttons because those seem important for CercubePlusExtra & uYouPlusExtra

**Comment by arichornlover** [comment]
Updated Cercube+ to v18.01.2!
I highly suggest updating because I gave you a ton many options to remove stuff y’all hate in the YouTube and some other helpful stuff to customize in Cercube+!

**Reply by arichornlover** [reply]
Updated Cercube+ to v18.01.4!
recommend updating if you're having some problems in v18.01.2 release.

**Reply by arichornlover** [reply]
Alright what the heck is going on every time I release a Cercube+ Version I always see google update the app now I don’t know if I should update Cercube+ to ‘v18.01.6‘ yet. Google must be messing with something in their app.

**Comment by arichornlover** [comment]
Hello users! I have released a new version of CercubePlus but only for testing purposes. If y’all want to contribute CercubePlus then navigate around the FLEX Menu. FLEX Menu can even help me even add & change the fonts around the App that were shown in the new previews in the Readme.md file.

**Comment by arichornlover** [comment]
to all of the CercubePlus iPad Users. in the next release I’ll add an Option called “iPhone Layout” so everyone can use features shown in the screenshots

![75D66ED3-D17B-4FCA-A21E-604BA72136C4](https://user-images.githubusercontent.com/78001398/215300725-ad3ec1ec-de2d-45d0-842f-0f70d0ba41ec.jpeg)
![A4D64B84-8202-440E-B91A-A0FCADCB3D84](https://user-images.githubusercontent.com/78001398/215300727-80ecf9b4-698f-4058-8b42-f3c2dce6c5f6.png)
and yes there will be unfixable issues if you do enable the iPhone Layout next release!

**Reply by arichornlover** [reply]
![A4016A20-9D5C-4B69-B1BB-DC500BA78488](https://user-images.githubusercontent.com/78001398/215391549-e80ae4cc-e6ae-4afa-9085-c5097700988e.png)

Another preview!
LowContrastMode updated to v1.2.1 with Live Chat Support!
Live Chat text is now Gray instead of White! This one took awhile to add though so that’s why I’m happy it worked.

**Reply by arichornlover** [reply]
![F335D55E-9784-4B04-97E1-839422EB64E8](https://user-images.githubusercontent.com/78001398/216804904-772f73c5-faf8-4385-8d72-745252bdadac.png)
CercubePlus v18.04.3-5.3.11’s new addition! iPhone Layout! It is shown in split-view mode on the left of my screen because using the option iPhone Layout in full screen does not look too good.

**Comment by arichornlover** [comment]
Unfortunately Majd deleted all of the Cercube Debs on his website. Don’t worry I’ve downloaded these recently before they were removed. Glad I can still improve CercubePlusExtra.
![418D604D-106C-477F-9CDB-962F9B8D5A6D](https://user-images.githubusercontent.com/78001398/219506713-5dc1984c-3251-45f1-b485-d1ff71782da8.jpeg)

**Reply by arichornlover** [reply]
alright y'all I'm doing something wrong and nothing is building the way I wanted it to be.
if it keeps failing and I'll try to keep fixing it. it won't be too long before I really Archive it. this is awful and I'm sorry that this is happening y'all.

**Comment by arichornlover** [comment]
To anyone using cercubeplus, I’ll be releasing a new version soon, I’m about to revamp Cercube+ with the same additions from uYouPlus. A ton of files may be changed, I’m here to let you know about these changes so you won’t be confused on why everything completely changed

**Reply by arichornlover** [reply]
CercubePlusExtra Release v18.13.2-5.3.11 has been deleted
Reason: cannot sign in at all when restarting app wait until the next release v18.13.4-5.3.11 to fix this problem.

**Comment by arichornlover** [comment]
Hmm it seems like some r/sideloaded users are questioning the name of this tweak, so I will talk about why it’s called “CercubePlusExtra”
“CercubePlus” is a tweak made by qnblackcat and as of 2023 that repository is gone and then “CercubePlusExtra” became a thing in late 2022/early 2023 but why did I include the word “Extra” in the tweak? Well pretty much I included that word because it pretty much just indicates the new maintainer (me) of adding new features that were by me & by other maintainers in README.MD file and also adding features from uYou+ which are just notably extra features that weren’t added in regular “CercubePlus” before it’s deletion. I think this is how good I can explain why the tweak is called that and to anyone saying I could change the name, calling it and changing the name to “CercubePlus” wouldn’t be a good idea now because it can now confuse the community who uses this tweak and I do appreciate your support and contribution on this tweak!

**Comment by arichornlover** [comment]
Alright I did an oopsie once again. I’ll be releasing **v18.11.2-5.3.13-F3** later or any time for users to be able to sign in into the app. and I’m also sorry for the CercubePlus Section not appearing in settings on the latest release.

**Reply by arichornlover** [reply]
I’m sorry for the disappointment of the release of **v18.15.1-5.3.13** not much were changed, I was only focusing on updating the YouTube version of it with some small little tweaks I modified to make it better. the **v18.13.4-5.3.13** release was the biggest so that is probably why I barely did anything this time.

**Comment by arichornlover** [comment]
<h2>CercubePlus Announcement</h2>

HELLO!
I'm doing some changes around the source code of CercubePlus as you might've noticed. and the one thing y'all would know what I organized some of the code like LowContrastMode to a seperate file. I'm doing changes to help Contributors find what they are looking for easier. and also fixing one of the biggest issues in CercubePlus #4 & #42
but the fourth issue will not fix the cercube downloading problem. which is why I'm adding another tweak which will be YouTube Reborn. but can't add it just yet until I figure out the Cercube stuff first. I just hope I won't break my promise to all of the users here.
[youtube-reborn.tar.gz](https://github.com/arichorn/CercubePlusExtra/files/11338902/youtube-reborn.tar.gz)

**Comment by arichornlover** [comment]
To anyone noticing the new repository name. This repository was originally supposed to be deleted a couple of days ago but I just decided to rename it and rename all of the releases. Wanted to let y’all know that I was originally gonna remove this repo before I changed my mind.

**Comment by arichornlover** [comment]
The Repository is now getting the uYouPlus/uYouEnhanced Treatment which means I am adding the missing stuff like for example. New Makefile, new buildapp.yml & proper rootless support.
